README.md

```diff
@@ -1,3 +1,194 @@
+};
+```
+and sending it to the function below by defining a `customToolbarArray: [keymapper]` field during the `DistortableImageOverlay` object instantiation.
+
+```js
+function(map, position, el, styleClass, DOMString) {
+  var custom_toolbar = L.control({ position: position });
+
+  custom_toolbar.onAdd = function() {
+    var el_wrapper = L.DomUtil.create(el, styleClass);
+    el_wrapper.innerHTML = DOMString;
+    return el_wrapper;
+  };
+  custom_toolbar.addTo(map);
+}
+```
+
+## Usage
+
+```js
+// basic Leaflet map setup
+map = new L.map('map').setView([51.505, -0.09], 13);
+L.tileLayer('https://{s}.tiles.mapbox.com/v3/anishshah101.ipm9j6em/{z}/{x}/{y}.png', {
+  maxZoom: 18,
+  attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
+    '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
+    'Imagery © <a href="http://mapbox.com">Mapbox</a>',
+  id: 'examples.map-i86knfo3'
+}).addTo(map);
+
+// keymap guide
+var keymapper = {
+  position: "topright",
+  el: "div",
+  styleClass: "l-container",
+  DOMString:
+    "<b><center><h3>Keymappings</h3><center><hr/></center></center><ul><li>L: Lock overlay</li><li>O: Outline overlay</li><li>R: Rotate overlay</li><li>RR: Distort overlay</li><li>T: Transparent overlay&nbsp;&nbsp;&nbsp;&nbsp;</li><li>DEL: Remove overlay</li></ul></b>"
+};
+
+// create an image
+img = new L.DistortableImageOverlay("example.png", {
+  suppressToolbar: false,
+  customToolbarArray: [keymapper],
+  hidden_tools: [ToggleEditable, ToggleExport, RemoveOverlay],
+  corners: [
+    new L.latLng(51.52, -0.1),
+    new L.latLng(51.52, -0.14),
+    new L.latLng(51.5, -0.1),
+    new L.latLng(51.5, -0.14)
+  ],
+  fullResolutionSrc: "large.jpg"
+}).addTo(map);
+
+L.DomEvent.on(img._image, 'load', img.editing.enable, img.editing); // enable editing
+
+```
+
+****
+
+## Setup
+
+1. From the root directory, run `npm install` or `sudo npm install`
+2. Open examples/index.html in a browser
+
+## Contributing
+
+1. This project uses `grunt` to do a lot of things, including concatenate source files from /src/ to /DistortableImageOverlay.js. But you may need to install grunt-cli: `npm install -g grunt-cli` first.
+2. Run `grunt` in the root directory, and it will watch for changes and concatenate them on the fly.
+
+To build all files from `/src/` into the `/dist/` folder, run `grunt concat:dist`.
+
+****
+
+### Contributors
+
+* Anish Shah, [@anishshah101](https://github.com/anishshah101)
+* Justin Manley, [@manleyjster](https://github.com/manleyjster)
+* Jeff Warren [@jywarren](https://github.com/jywarren)
+
+Many more at https://github.com/publiclab/Leaflet.DistortableImage/graphs/contributors
+=======
```

Comment:
Oh, did this somehow get shown twice during a merge resolution? I think we need to delete what's below this line.